paddle/fluid/framework/ir/mkldnn/fc_act_fuse_pass_tester.cc

```diff
@@ -0,0 +1,254 @@
+///                              will have persistable attribute set to true.
+///
+/// @return     The program descriptor object.
+///
+ProgramDesc BuildProgramDesc(const std::vector<std::string>& transient_vars,
+                             const std::vector<std::string>& persistent_vars) {
+  ProgramDesc prog;
+
+  auto add_var_to_prog = [&prog](const std::string& var_name) -> VarDesc* {
+    auto var = prog.MutableBlock(0)->Var(var_name);
+    var->SetType(proto::VarType::LOD_TENSOR);
+    return var;
+  };
+
+  for (const auto& v : transient_vars) {
+    add_var_to_prog(v);
+  }
+
+  for (const auto& v : persistent_vars) {
+    auto* var = add_var_to_prog(v);
+    var->SetPersistable(true);
+  }
+
+  return prog;
+}
+
+///
+/// @brief      Execute pass on provided graph and perform checks.
+///
+/// @param      graph                The graph we run pass on.
+/// @param[in]  from                 The name of a 'starting' node sequence in a
+///                                  graph. This would be used to test for
+///                                  correct node connections.
+/// @param[in]  to                   The name of a 'ending' node sequence in a
+///                                  graph. This would be used to test for
+///                                  correct node connections.
+/// @param[in]  removed_nodes_count  The number of nodes we expect will be
+///                                  removed/fused after pass execution.
+/// @param[in]  added_nodes_count    The number of nodes we expect will be
+///                                  added after pass execution.
+///
+void RunPassAndAssert(Graph* graph, const std::string& from,
+                      const std::string& to, int removed_nodes_count,
+                      int added_nodes_count = 0) {
+  EXPECT_TRUE(TestIsReachable(*graph, from, to));
+  int original_nodes_num = graph->Nodes().size();
+  auto pass = PassRegistry::Instance().Get("fc_act_fuse_pass");
+  pass->Apply(graph);
+  int current_nodes_num = graph->Nodes().size();
+
+  EXPECT_TRUE(TestIsReachable(*graph, from, to));
+  EXPECT_EQ(original_nodes_num - removed_nodes_count + added_nodes_count,
+            current_nodes_num);
+}
+
+}  // namespace
+
+// ------------------------------ Test cases -----------------------------------
+
+TEST(FuseFCActOneDNNPass, ThrowUseMkldnn) {
+  auto prog = BuildProgramDesc({"x", "fc_y", "act_y"}, {"weights", "bias"});
+  CreateOp(&prog, "fc",
+           {
+               {"Input", "x"}, {"Weights", "weights"}, {"Bias", "bias"},
+           },
+           {{"Out", "fc_y"}}, false);
+  CreateOp(&prog, "gelu", {{"Input", "fc_y"}}, {{"Out", "act_y"}}, false);
+
+  Graph graph(prog);
+  // No fusion in this attribute configuration
+  constexpr int removed_nodes_count = 0;
+
+  EXPECT_THROW(RunPassAndAssert(&graph, "x", "act_y", removed_nodes_count),
+               paddle::platform::EnforceNotMet);
+}
+
```

Comment:
Please add:
*  _positive_ test cases and check correct fusing behaviour for all available activations {"gelu", "tanh", "gelu_erf", gelu_tanh"}. 
* checks for op attribute values correctness. (you're setting `activation_type` attribute in fuse pass)